CalibPPS/ESProducers/plugins/PrintTotemDAQMapping.cc

```diff
@@ -30,33 +31,45 @@ class PrintTotemDAQMapping : public edm::one::EDAnalyzer<> {
 private:
   /// label of the CTPPS sub-system
   std::string subSystemName;
-  edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken_;
-  edm::ESGetToken<TotemAnalysisMask, TotemReadoutRcd> maskToken_;
+  edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken;
+  edm::ESGetToken<TotemAnalysisMask, TotemAnalysisMaskRcd> maskToken;
   void analyze(const edm::Event &e, const edm::EventSetup &es) override;
 };
 
 PrintTotemDAQMapping::PrintTotemDAQMapping(const edm::ParameterSet &ps)
     : subSystemName(ps.getUntrackedParameter<std::string>("subSystem")),
-      mappingToken_(esConsumes(edm::ESInputTag("", subSystemName))),
-      maskToken_(esConsumes(edm::ESInputTag("", subSystemName))) {}
+      mappingToken(esConsumes(edm::ESInputTag("", subSystemName))),
+      maskToken(esConsumes(edm::ESInputTag("", subSystemName))) {}
 
 //----------------------------------------------------------------------------------------------------
 
 void PrintTotemDAQMapping::analyze(const edm::Event &, edm::EventSetup const &es) {
   // get mapping
-  auto const &mapping = es.getData(mappingToken_);
+  if (auto mappingHandle = es.getHandle(mappingToken)) {
+    auto const &mapping = *mappingHandle;
+    // print mapping
+    for (const auto &p : mapping.VFATMapping)
+      edm::LogInfo("PrintTotemDAQMapping mapping") << "    " << p.first << " -> " << p.second;
+
+    for (const auto &p : mapping.totemTimingChannelMap)
+      edm::LogInfo("PrintTotemDAQMapping channel mapping") << "    " << p.first << " plane " << p.second.plane << " channel " << p.second.channel;
```

Comment:
After addressing the comment in `WriteCTPPSTotemDAQMappingMask.cc` about printing you could use the overloaded `<<` operators here.

Some more places like this in this file